.travis.yml

```diff
@@ -0,0 +1,19 @@
+# Force OS X.
+language: objective-c
+os: osx
+osx_image: xcode7 # 10.10
+matrix:
+  include:
+    - osx_image: beta-xcode6.2 # 10.9
+
+rvm:
+  - rvm use system
+
+install:
+  - rvm use system
+  - cd /usr/local; git fetch --unshallow
+  - cd /usr/local; [ "$TRAVIS_PULL_REQUEST" != "false" ] && git fetch origin +refs/pull/$TRAVIS_PULL_REQUEST/merge
+  - cd /usr/local; git reset --hard $TRAVIS_COMMIT
```

Comment:
I don't agree. If we do anything it should be to symlink the `.git` directory. The symlinked `brew` setup is not what most of our users do.